executor/aggfuncs/func_value_test.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aggfuncs_test
+
+import (
+	. "github.com/pingcap/check"
+	"github.com/pingcap/parser/ast"
+	"github.com/pingcap/parser/mysql"
+	"github.com/pingcap/tidb/executor/aggfuncs"
+	"github.com/pingcap/tidb/types"
+	"github.com/pingcap/tidb/util/chunk"
+)
+
+func getEvaluatedMemDelta(row *chunk.Row, dataType *types.FieldType) (memDelta int64) {
+	memDelta = 0
```

Comment:
I think we don't need this. the memDelta is always init to 0, 
see https://play.studygolang.com/p/j-tmF88zLLx

Reply:
Yeah, sure!